tests/packagedcode/data/rubygems/gemspec/rubocop.gemspec

```diff
@@ -11,12 +11,9 @@ Gem::Specification.new do |s|
   s.platform = Gem::Platform::RUBY
   s.required_ruby_version = '>= 2.2.0'
   s.authors = ['Bozhidar Batsov', 'Jonas Arvidsson', 'Yuji Nakayama']
-  s.description = <<-DESCRIPTION
-    Automatic Ruby code style checking tool.
-    Aims to enforce the community-driven Ruby Style Guide.
-  DESCRIPTION
+  s.description = "Automatic Ruby code style checking tool. Aims to enforce the community-driven Ruby Style Guide."
 
-  s.email = 'rubocop@googlegroups.com'
+  s.email = ['rubocop@googlegroups.com']
```

Comment:
Why did you change this test file? This is a real one at https://github.com/rubocop-hq/rubocop/blob/5cb29efb62f3ff36cbc9ad75d2c29392737734e2/rubocop.gemspec
We need to handle it, and not change it to fit the code ;)